core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb.go

```diff
@@ -370,15 +375,14 @@ func (vdb *VersionedDB) ProcessIndexesForChaincodeDeploy(namespace string, index
 	}
 	sort.Strings(indexFilesName)
 	for _, fileName := range indexFilesName {
-		_, err = db.createIndex(string(indexFilesData[fileName]))
-		switch {
-		case err != nil:
+		if _, err = db.createIndex(string(indexFilesData[fileName])); err != nil {
 			logger.Errorf("error creating index from file [%s] for chaincode [%s] on channel [%s]: %+v",
 				fileName, namespace, vdb.chainName, err)
-		default:
-			logger.Infof("successfully submitted index creation request present in the file [%s] for chaincode [%s] on channel [%s]",
-				fileName, namespace, vdb.chainName)
+			continue
```

Comment:
Since you are just streamlining the control flow, let me ask: is the `continue` better than just encasing the logger call below in an else?

Reply:
reverted, since i don't have a strong opinion here